### DISCUSSION TITLE
Scheduled Job - get job time on subscribe handler

### DISCUSSION THREAD

**Discussion by ItaiHuber** [timgit/pg-boss#298] [Q&A]
I have an hourly schedule, i need to support one of these two options

Option 1 - The called callback method will receive the time (and optionaly date) of this schedule.
So lets say my process was down for 5 hours, and i have a scehduler for every hour, when it gets up, i will get 5 notification callbacks called, i want to know for each one which hour it was suppose to run on.

Option 2 - If process is down for 5 hours, get only 1 call (skip on the missed schedules)

**Comment by timgit** [comment]
The scheduler works just like crontab. If the system is off it will not send jobs. This is your option 2

**Comment by ItaiHuber** [comment]
It’s not what I experienced 
I set a crontab to every minute
Stopped my service
Started it again and got several jobs one after the other 


Itai Huber
